src/client/src/admin.rs

```diff
@@ -44,6 +47,69 @@ impl FluvioAdmin {
         Self(client)
     }
 
+    /// Creates a new admin connection using default configurations
+    ///
+    /// This will attempt to read a Fluvio cluster configuration from
+    /// your `~/.fluvio/config` file, or create one with default settings
+    /// if you don't have one. If you want to specify a configuration,
+    /// see [`connect_with_config`] instead.
+    ///
+    /// The admin interface requires you to have administrator privileges
+    /// on the cluster which you are connecting to. If you don't have the
+    /// appropriate privileges, this connection will fail.
+    ///
+    /// # Example
+    ///
+    /// ```no_run
+    /// # use fluvio::{FluvioAdmin, FluvioError};
+    /// # async fn do_connect() -> Result<(), FluvioError> {
+    /// let admin = FluvioAdmin::connect().await?;
+    /// # Ok(())
+    /// # }
+    /// ```
+    ///
+    /// [`connect_with_config`]: ./struct.FluvioAdmin.html#method.connect_with_config
+    pub async fn connect() -> Result<Self, FluvioError> {
+        let config_file = ConfigFile::load_default_or_new()?;
+        let cluster_config = config_file
+            .config()
+            .current_cluster()
+            .ok_or_else(|| FluvioError::ConfigError("failed to load cluster config".to_string()))?;
+        Self::connect_with_config(cluster_config).await
+    }
+
+    /// Creates a new admin connection using custom configurations
+    ///
+    /// The admin interface requires you to have administrator privileges
+    /// on the cluster which you are connecting to. If you don't have the
+    /// appropriate privileges, this connection will fail.
+    ///
+    /// # Example
+    ///
+    /// ```no_run
+    /// # use fluvio::{FluvioAdmin, FluvioError};
+    /// use fluvio::config::ConfigFile;
+    /// #  async fn do_connect_with_config() -> Result<(), FluvioError> {
+    /// let config_file = ConfigFile::load_default_or_new()?;
+    /// let fluvio_config = config_file.config().current_cluster().unwrap();
+    /// let admin = FluvioAdmin::connect_with_config(fluvio_config).await?;
+    /// # Ok(())
+    /// # }
+    /// ```
+    pub async fn connect_with_config(config: &FluvioConfig) -> Result<Self, FluvioError> {
```

Comment:
not show stopper for now. but codes are duplicated.  maybe this should be turned into connect trait.